src/invalset-vixl.h

```diff
@@ -112,7 +112,7 @@ class InvalSet {
   size_t size() const;
 
   // Returns true if no elements are stored in the set.
-  // Note that this does not mean the the backing storage is empty: it can still
+  // Note that this does not mean the backing storage is empty: it can still
```

Comment:
I'm still experimenting with this heuristic, but I'm fairly confident about this specific correction